lib/signaling/v2/peerconnection.js

```diff
@@ -126,6 +130,12 @@ function PeerConnectionV2(id, encodingParameters, preferredCodecs, options) {
         ? options.log.createLog('signaling', this)
         : null
     },
+    _mediaSenders: {
```

Comment:
Can we let this be `_mediaTrackSenders`?

Reply:
@markandrus We actually store `RTCRtpSender`s in this collection. So isn't this a more appropriate name?

Reply:
OK, can we call this `_rtpSenders` then?